private/coursier_utilities.bzl

```diff
@@ -28,18 +28,49 @@ SUPPORTED_PACKAGING_TYPES = [
 ]
 
 def strip_packaging_and_classifier(coord):
-    # We add "pom" into _COURSIER_PACKAGING_TYPES here because "pom" is not a
-    # packaging type that Coursier CLI accepts.
-    for packaging_type in SUPPORTED_PACKAGING_TYPES + ["pom"]:
-        coord = coord.replace(":%s:" % packaging_type, ":")
-    for classifier_type in ["sources", "natives"]:
-        coord = coord.replace(":%s:" % classifier_type, ":")
+    # Strip some packaging and classifier values based on the following maven coordinate formats
+    # groupId:artifactId:version
+    # groupId:artifactId:packaging:version
+    # groupId:artifactId:packaging:classifier:version
+    coordinates = coord.split(":")
+    if len(coordinates) > 4:
+        if coordinates[3] in ["sources", "natives"]:
```

Comment:
I think `natives` should just be `native` but I didn't want to change it in this PR.

Reply:
SGTM